commands.adoc

```diff
@@ -223,10 +223,18 @@ Similarly, you can use `$ vagrant service-manager status openshift`,
 
 If you need to install the client binary for a specified service, use: +
 
-`vagrant service-manager install-cli [service]` +
+`vagrant service-manager install-cli [service] [options] [arg]` +
 
 The possible options for service are `docker` and `openshift`. +
 
+The following options can be used: +
+
+`--cli-version`::  Version of binary to install +
+`--path`::   Absolute or relative path where to install the binary +
```

Comment:
Yes, Specify the version of the binary to be installed. and Specify the absolute or relative path where the binary should be installed.